src/core/varinfo.jl

```diff
@@ -310,8 +318,23 @@ end
 # Rand & replaying method for VarInfo #
 #######################################
 
-# Check if a vn is set to NULL
-isnan(vi::VarInfo, vn::VarName) = any(isnan.(getval(vi, vn)))
+check_flag(vi::VarInfo, vn::VarName, flag::String) = vi.flags[flag][getidx(vi, vn)]
```

Comment:
`get_flag` instead ?

Reply:
Or maybe `is_flagged`? I was trying to find a name which fits the use case of this function. As it always return a `Bool` type, we always use it like 
```julia
if is_flagged(vi, vn, "missing")
  ...
end
```